src/posts/lucky_0.26.cr

```diff
@@ -0,0 +1,205 @@
+      end
+    end
+
+    # src/actions/repos/delete.cr
+    class Repos::Delete < BrowserAction
+      delete "/repos/:repo_id" do
+        repo = RepoQuery.find(repo_id)
+
+        DeleteRepo.destroy(repo) do |operation, deleted_repo|
+          if operation.deleted?
+            flash.success = "The repo \#{deleted_repo.git_name} has been deleted"
+            redirect to: Home::Index
+          else
+            TimeBomb.start_countdown(10.seconds)
+          end
+        end
+      end
+    end
+    ```
+
+    [Read more](https://github.com/luckyframework/avram/pull/573) on this feature.
+
+    ### CIText Support
+
+    This was new to some of us that had never even heard of "citext" before. The "citext" column in postgres is "Case-Insensitive Text". This allows you to
+    store a string in your database, and do lookups against it without the need to do something like `"WHERE LOWER(email) = ?", email.downcase`. Instead, your
+    users could spell their email like "LucKyDawG@gmAIl.com" and still log in with "luckydawg@gmail.com".
+
+    By default, all `String` columns in Avram are case sensitive, but if you'd like to use this citext column, just add `case_sensitive: false` to your migrations!
+
+    ```crystal
+    # NOTE: You must enable the extension for this feature to work!
+    enable_extension "citext"
+
+    create table_for(User) do
+      add name : String
+      add email : String, case_sensitive: false
+    end
+    ```
+
+    [Read more](https://github.com/luckyframework/avram/pull/608) on this feature.
+
+    ### Composite Keys
+
+    Speaking of migrations... We're slowly working on adding in "composite-primary keys". This is the concept of having 2 (or more) columns that are combined for primary key
+    lookups in your database. Generally you'll just use a single `id` column, but with a composite primary key, you could have "id1" and "id2", and a record is looked up
+    using both of those fields.
+
+    For now, we've only enabled this on the migration side. [See the PR](https://github.com/luckyframework/avram/pull/616).
+
+    ### Ditching `route` and `nested_route`
+
+    Lucky comes built with two special macros `route` and `nested_route` which automatically define both the HTTP Verb, and the actual request path for your actions.
+    This is nice because `Users::Index` magically becomes `get "/users"`, and `Users::Comments::Index` is turned in to `get "/users/:user_id/comments"`. It cleans
+    up your code a little, and makes things consistent.
+
+    The downside to this is, what does `Admin::Report::Spending` become, or how about `Git::Push`? The `route` and `nested_route` macros work great when you're
+    building simple REST related resources, but in more complex cases, you end up writing your routes out manually. This means your actions are no longer written consistently.
+    Also, newcomers that aren't familiar with what "REST" is may not understand the "magic" that is happening.
```

Comment:
Thoughts on deleting this sentence? I'm thinking that most people coming to a web framework will know what REST is. Or at least, users not knowing what the HTTP verbs are feels like a reason to _not_ make this change.

Reply:
Sure, I'm cool with that.